nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -365,7 +365,6 @@ let
       preferLocalBuild = true;
     }
     ''
-      cat ${php}/etc/php.ini > $out
```

Comment:
I would like to see a diff of the file if you would add >> and I believe that this line is important as https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/web-servers/phpfpm/default.nix#L31 `cat` both files

Reply:
At least I think this needs a cleanup somewhere somehow.